app/javascript/pages/country/widget/widgets/widget-fao-cover/widget-fao-cover.js

```diff
@@ -15,7 +14,7 @@ const mapStateToProps = ({ widgetFAOCover }, ownProps) => {
   const selectorData = {
     data,
     locationNames,
-    colors: COLORS.extent
+    colors: ownProps.colors.extent
```

Comment:
In some cases you are sending `ownProps.colors` to the selector, but in other cases you are sending the specific category, like `ownProps.colors.extent`. Is that ok?

Reply:
Yes that is correct. The goal was to use the widget type to dynamically send the colours to all widgets. In some cases the widget type doesn't line up as it is fao which is needed for the whitelist. Maybe you try to think of a better a solution. Or maybe it is a redundant PR.

Reply:
oh ok! I understand now :)
Looks good to me! good job.